automates/text_reading/src/main/scala/org/clulab/utils/MentionUtils.scala

```diff
@@ -0,0 +1,116 @@
+package org.clulab.utils
+
+import org.clulab.aske.automates.OdinEngine
+
+import java.io.{File, IOException}
+import org.clulab.aske.automates.attachments.{AutomatesAttachment, MentionLocationAttachment}
+import org.clulab.odin.{EventMention, Mention}
+import org.clulab.aske.automates.apps._
+import org.clulab.aske.automates.mentions.CrossSentenceEventMention
+import org.clulab.processors.fastnlp.FastNLPProcessor
+import org.slf4j.{Logger, LoggerFactory}
+
+import scala.collection.mutable.ArrayBuffer
+
+
+object MentionUtils {
+  /**stores methods related to text mention extraction*/
+  protected lazy val logger: Logger = LoggerFactory.getLogger(this.getClass)
+  lazy val proc = new FastNLPProcessor()
+
+  def getMentionText(mention: Mention): String = {
+
+    // get text bound mention text taking into account the presence of discontinuous character offset attachment
+    // fixme: add functionality to get text of an event mention whose arguments have discont attachments
```

Comment:
Did you already address this fixme, or is it for future reference? If you did, please delete the note.